kamelet-support/src/test/java/io/kaoto/backend/metadata/parser/step/kamelet/KameletParseCatalogTest.java

```diff
@@ -155,6 +172,36 @@ void testCluster() {
         assertEquals(1, steps.size());
     }
 
+    //reproducer for issue #656
+    @Test
+    void testClusterWithWholeKamelets() throws IOException {
+        int numberOfKamelets = 0;
+
+        try (CloseableHttpClient client = HttpClients.createDefault()) {
+            HttpGet request =
```

Comment:
Can't we use the public link instead? https://github.com/apache/camel-kamelets/archive/refs/tags/v3.20.4.zip

Reply:
Yes, but after that, the test needs to extract files and loads them via Kubernetes client.